scripts/game.py

```diff
@@ -0,0 +1,83 @@
+#! /usr/bin/env python
```

Comment:
Can we rename this abstract?  Maybe `roomba_controller_test.py`?

Reply:
fixed